fourkitchens/suhumsci
# SHS-4922: update postcard and spotlight slider paragraphs thumbnails

# READY FOR REVIEW

## Summary
- Update Postcard and Spotlight Slider (now called "Spotlight(s)") paragraph browser thumbnails

## Steps to Test
1. Create a Flexible page
2. Click on the "Add Component" button in the Main Region. Confirm that the thumbnails for the "Postcard" and "Spotlight(s)" paragraphs match the images below

### Postcard
![A Flexible Page _ Center for African Studies (1)](https://github.com/fourkitchens/suhumsci/assets/3496574/642df27f-b846-44c7-a508-3e3133a4e97c)

### Spotlight(s)
![Spotlight2](https://github.com/fourkitchens/suhumsci/assets/3496574/ac2a5818-dcc4-453e-8778-a7ea06bdff51)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
